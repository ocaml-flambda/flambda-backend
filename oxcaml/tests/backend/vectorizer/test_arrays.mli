(* blank, make sure all the functions are called from top-level *)
