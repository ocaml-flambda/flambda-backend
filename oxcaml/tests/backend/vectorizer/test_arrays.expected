add_arrays_unrolled_manually 17 18 19 20 21 22 23 24 25 26 
add_arrays_unrolled_safe 17 18 19 20 21 22 23 24 25 26 
add_arrays_rec_unrolled_attribute 17 18 19 20 21 22 23 24 25 26 
add_arrays_for 17 18 19 20 21 22 23 24 25 26 
add_arrays_rec 17 18 19 20 21 22 23 24 25 26 
initialize_array_const_unrolled_manually 0 0 0 0 0 0 0 0 0 0 
initialize_arrays_const_unrolled_manually 0 0 0 0 0 0 0 0 0 0 
initialize_array_unrolled_manually 17 17 17 17 17 17 17 17 17 17 
initialize_floatarray_unrolled_manually 7.700000 7.700000 7.700000 7.700000 7.700000 7.700000 7.700000 7.700000 7.700000 7.700000 
